### ISSUE TITLE
Bot crashes after one notification: "unexpected EOF"

### ISSUE THREAD

**Issue by TKussel** [scottleedavis/mattermost-plugin-remind#64]
After one notification the bot crashes and refuses any further command. Neither new ```/remind``` commands, nor the buttons in the notification work. Here's a snippet of the log file. It starts with the creation of a test one minute reminder:
```
{"level":"info","ts":1555512140.5666263,"caller":"go-plugin/stream.go:15","msg":"{\"level\":\"info\",\"msg\":\"found the files?\"}","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind","source":"plugin_stdout"}
{"level":"error","ts":1555512156.886507,"caller":"mlog/sugar.go:23","msg":"new reminder <user>","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind"}
{"level":"info","ts":1555512168.548439,"caller":"mlog/sugar.go:19","msg":"[in 1 minute]","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind"}

{"level":"error","ts":1555512235.1459863,"caller":"mlog/log.go:172","msg":"Plugin failed to ServeHTTP, RPC call failed","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind","error":"unexpected EOF"}
{"level":"error","ts":1555512235.1461823,"caller":"mlog/log.go:172","msg":"Action integration error","path":"/api/v4/posts/fehjrwrxp3d1pdd65ixnf1b3dw/actions/6ogwqjnb8tboffkguhp4qnhjga","request_id":"zh6bq66ghff9iyxn7gsf18asar","ip_addr":"<ip>","user_id":"<u_id>","method":"POST","err_where":"DoActionRequest","http_code":400,"err_details":"status=500"}
{"level":"error","ts":1555512238.9018447,"caller":"mlog/log.go:172","msg":"Plugin failed to ServeHTTP, RPC call failed","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind","error":"connection is shut down"}
{"level":"error","ts":1555512244.5755908,"caller":"mlog/log.go:172","msg":"Plugin failed to ServeHTTP, muxBroker couldn't Accept request body connection","plugin_id":"com.github.scottleedavis.mattermost-plugin-remind","error":"timeout waiting for accept"}```
Deactivating and reactivating the plugin in the system console works (for the next, single reminder).

**Comment by hanzei** [comment]
What server version are you using @TKussel?

**Comment by scottleedavis** [comment]
@TKussel  could you try the latest release?
https://github.com/scottleedavis/mattermost-plugin-remind/releases/tag/v0.1.3

Could you also report the output from the following command before doing reminders?
`/remind __user`
It should display the locale and location of the current user if it exists.  (problems if they don't)

**Comment by TKussel** [comment]
I tried v0.1.13 under Debian stretch (i.e. server dist plugin-linux-amd64)
```/remind __user``` outputs ```en Local``` and the problem persists in v0.1.13

**Comment by scottleedavis** [comment]
@TKussel I was able to reproduce the behavior on debian stretch.

<img width="1665" alt="Screen Shot 2019-04-19 at 8 09 02 AM" src="https://user-images.githubusercontent.com/2914443/56431035-5780be80-627d-11e9-8337-6921c2e427cc.png">

I Then set siteURL in configuration.  After that, the plugin runs as expected.

![debian-stretch](https://user-images.githubusercontent.com/2914443/56431640-53ee3700-627f-11e9-8948-ad8716525fba.gif)

Is the siteURL set on your system?  If not, does this solve the issue?

If siteURL is already set, could you please try `/remind __clear` and try setting a reminder again?

**Comment by scottleedavis** [comment]
Reading through your original issue, I don't think I have replicated it yet.

**Comment by scottleedavis** [comment]
@TKussel I was able to replicate the situation on debian stretch, where the triggered reminder wouldn't happen and wasn't been saved properly leaving things in an incorrect state.  released a new version v0.1.5 , hoping that solves it for you.  If not lets keep at it.  :)

**Comment by TKussel** [comment]
@scottleedavis Thanks fo all your effort! Silly me had still a temporary siteURL in the configuration. Correcting that fixed my issue!
